crowbar_framework/app/models/ntp_service.rb

```diff
@@ -110,8 +110,9 @@ def skip_unchanged_node?(node_name, old_role, new_role)
     # if the node changed roles, then we need to apply
     return false if node_changed_roles?(node_name, old_role, new_role)
```

Comment:
We'd also need a check as suggested in https://github.com/crowbar/crowbar-core/pull/1420#issuecomment-342800525

Reply:
Working on that